specs/windows/programs.table

```diff
@@ -1,9 +1,10 @@
-table_name("programs", aliases=["programs_and_features"])
-description("Rrepresents products as they are installed by Windows Installer. A product generally correlates to one installation package.")
+table_name("programs", aliases=["programs_and_features", "apps", "applications"])
```

Comment:
Be careful, `apps` is used by macOS. I'd recommend keeping this called `programs` or `programs_and_features` unless there's parity with macOS's application table.

Reply:
The fact that this passed tests is concerning, maybe we're not checking aliases when `--enable_foreign` is used.